dvc/command/experiments/show.py

```diff
@@ -472,20 +472,20 @@ def show_experiments(
         }
     )
 
-    if kwargs.get("only_changed", False) or html:
+    if kwargs.get("only_changed", False) or pcp:
         td.drop_duplicates("cols")
 
-    html_args = {}
-    if html:
+    pcp_args = {}
+    if pcp:
         td.dropna("rows", how="all")
         td.column("Experiment")[:] = [
             # remove tree characters
             str(x).encode("ascii", "ignore").strip().decode()
             for x in td.column("Experiment")
         ]
         out = kwargs.get("out") or "dvc_plots"
-        html_args["output_path"] = (Path.cwd() / out).resolve()
-        html_args["color_by"] = kwargs.get("sort_by") or "Experiment"
+        pcp_args["output_path"] = (Path.cwd() / out).resolve()
+        pcp_args["color_by"] = kwargs.get("sort_by") or "Experiment"
 
     td.render(
```

Comment:
This `table.render()` does not make sense for rendering anything other than a table. I'd keep these separate and just use `td.to_parallel_coordinates` helper function directly here instead of extending `render`.